homeassistant/components/tplink_omada/config_flow.py

```diff
@@ -222,3 +236,104 @@ async def _test_login(
             _LOGGER.exception("Unexpected exception")
             errors["base"] = "unknown"
         return None
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlow:
+        """Create the options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(OptionsFlowWithConfigEntry):
```

Comment:
We shouldn't need an options flow since device tracker scanner entities are disabled by default.

Reply:
Yeah, I didn't really want to flood the list with disabled entities, and I wanted to be able to choose what type of tracking to do. I could remove the flow, but I sort of prefer to choose in advance which entities to track. Also, I'm going to add a poll frequency option for device tracking. So I'd rather keep this if that's ok?

Reply:
No, we've made the device tracker entities disabled by default so we don't need any options.

Poll frequence options is also not allowed. The user can turn of polling for config entries and automate it themselves with the `homeassistant.update_entity` service.

https://www.home-assistant.io/common-tasks/general/#defining-a-custom-polling-interval

Reply:
OK, I'll remove the options flow too... I'm a bit confused how the manual refresh polling automation plays with the Update coordinator, which updates the whole list of entities. Would the user just select one entity and that would have the side-effect of updating them all? How would they know to do that?
I assume the update coordinator is clever enough to not poll if there are no enabled entities listening to it, even if I call the initial load method, and also clever enough to not poll twice if two entities ask for a refresh at almost the same time.

Reply:
It's ok to add a section to the docs of how to automate the polling if you think that's needed.

The first explicit refresh by the integration will fetch data.

There won't be any recurring refresh by default if there are no listeners.

There's a default refresh debounce of ten seconds in the coordinator. It means that if there was a refresh, any subsequent refresh within ten seconds will not be executed but scheduled to run after the ten seconds have passed, and only one refresh will be scheduled.

Eg if three requests for refresh come in during the debounce time, only one refresh will be scheduled to run after the debounce time expires. The debounce can be customized by the coordinator. See the debounce helper for details.